features/leaveroom/impl/src/main/kotlin/io/element/android/features/leaveroom/impl/LeaveRoomPresenter.kt

```diff
@@ -88,20 +85,18 @@ private suspend fun showLeaveRoomAlert(
 
 private suspend fun MatrixClient.leaveRoom(
     roomId: RoomId,
-    roomMembershipObserver: RoomMembershipObserver,
     confirmation: MutableState<LeaveRoomState.Confirmation>,
     progress: MutableState<LeaveRoomState.Progress>,
     error: MutableState<LeaveRoomState.Error>,
 ) {
     confirmation.value = LeaveRoomState.Confirmation.Hidden
     progress.value = LeaveRoomState.Progress.Shown
     getRoom(roomId)?.use { room ->
-        room.leave().onSuccess {
-            roomMembershipObserver.notifyUserLeftRoom(room.roomId)
-        }.onFailure {
-            Timber.e(it, "Error while leaving room ${room.displayName} - ${room.roomId}")
-            error.value = LeaveRoomState.Error.Shown
-        }
+        room.leave()
+            .onFailure {
+                Timber.e(it, "Error while leaving room ${room.displayName} - ${room.roomId}")
```

Comment:
Not new but we should only log the `roomId` for privacy reason.

Reply:
Changed